lib/kdl/include/kdl/result_io.h

```diff
@@ -33,11 +33,11 @@ namespace kdl {
     template <typename... Errors>
     std::ostream& operator<<(std::ostream& str, const result<void, Errors...>& result) {
         result.visit(kdl::overload {
-            []()               { str << "void"; },
```

Comment:
Oh, this file had compile errors, but it was being skipped due to the error with the guard?

Reply:
Yeah! Goes to show how horrific such errors are. We could switch to pragma once to avoid this. I think it's understood by all compilers we use.

Reply:
Yeah, pragma once sounds good